MatrixSDK/MXRestClient.m

```diff
@@ -1406,10 +1411,69 @@ - (MXHTTPOperation*)requestTokenFromEndpoint2:(NSString *)path
 - (MXHTTPOperation*)addIdentityAccessTokenToParameters:(NSDictionary *)parameters
                                                success:(void (^)(NSDictionary *updatedParameters))success
                                                failure:(void (^)(NSError *error))failure
+{
+    MXHTTPOperation *operation = [self useIdentityAccessToken:^(NSString * _Nullable accessToken) {
```

Comment:
I would perhaps rename the method to something like `useIdentityAccessTokenIfNeccessary`, because from this code missing `accessToken` looks like it should be an error flow, despite the `_Nullable` annotations